charts/fluid-dataprocess/common/templates/job.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ printf "%s-job" .Release.Name }}
+  labels:
+    release: {{ .Release.Name }}
+    role: dataprocess-job
+    app: fluid-dataprocess
+    targetDataset: {{ required "targetDataset should be set" .Values.dataProcess.targetDataset }}
+spec:
```

Comment:
add `ownerReferences` like DataLoad/DataMigrate?

Reply:
DataProcess is user-defined customized code logic to process data. I think maybe adding ownerReference will automatically delete its job when dataset is deleted. This is not good if users want to keep their data processing job.

Reply:
How about adding dataProcessCleanPolicy? By default, it's OnDatasetDeleted. And other values are 'None'.

Or ttl like BatchJob? WDYT?

Reply:
Agreed with OnDatasetDeleted by default.

Reply:
@xliuqq Yes. But I don't think it's a expected behavior that users need. For example, they launch a model training job by submitting a DataProcess. It is possible users may keep their model training job's logs. `OnDatasetDeleted` is a good idea, I will support it in future PR.